src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Block.cs

```diff
@@ -42,6 +38,58 @@ public override BoundNode VisitBlock(BoundBlock node)
             return new BoundBlock(node.Syntax, synthesizedLocal == null ? node.Locals : node.Locals.Add(synthesizedLocal), node.LocalFunctions, builder.ToImmutableAndFree(), node.HasErrors);
         }
 
+
+        /// <summary>
+        /// Visit a partial list of statements that possibly contain using declarations
+        /// </summary>
+        /// <param name="statements">The list of statements to visit</param>
+        /// <param name="startIndex">The index of the <paramref name="statements"/> to begin visiting at</param>
+        /// <returns>An <see cref="ImmutableArray{T}"/> of <see cref="BoundStatement"/></returns>
+        public ImmutableArray<BoundStatement> VisitStatementSubList(ImmutableArray<BoundStatement> statements, int startIndex = 0)
+        {
+            var builder = ArrayBuilder<BoundStatement>.GetInstance();
+            for (int i = startIndex; i < statements.Length; i++)
+            {
+                BoundStatement statement = (BoundStatement)VisitPossibleUsingDeclaration(statements[i], statements, i, out var replacedUsingDeclarations);
+                if (statement != null)
```

Comment:
>statement != null [](start = 20, length = 17)

Is the result ever `null`?

Reply:
The previous code called Visit(node) and checked for null. We eventually call down this path for non-labeled or using statements, so I believe it's possible.